src/evo/cbtx.cpp

```diff
@@ -448,7 +449,7 @@ std::string CCbTx::ToString() const
         creditPoolBalance / COIN, creditPoolBalance % COIN);
 }
 
-std::optional<CCbTx> GetCoinbaseTx(const CBlockIndex* pindex)
+std::optional<std::pair<CBLSSignature, uint32_t>> GetNonNullCoinbaseChainlock(const CBlockIndex* pindex)
```

Comment:
6697ad1202d745521ee6b8ec0e931811ae522519: commit title should say smth like "drop GetCoinbaseTx", not "make GetCoinbaseTx static"